client/index.html

```diff
@@ -1457,5 +1748,8 @@ <h3 translation="partners"></h3>
 
     <!-- JS -->
     <script src="./src/index.ts" type="module"></script>
+    <script src="./src/loginBtn.ts" type="module"></script>
+    <script src="./src/accountVisibility.ts" type="module"></script>
+    <script src="./src/myStats.ts" type="module"></script>
```

Comment:
you only need 1 entrypoint, cuts down on your 5 document event listeners as well